### PR TITLE
WINUI-3035_UG_document_for_auto-append

### PR DESCRIPTION
**Pull request by YuvarajMasilamani**
1. UG document for auto-append in AutoComplete.
2. UG document for auto-append in ComboBox.

### CODE REVIEW

File: winui/AutoComplete/Selection.md [hunk 1]
```diff
@@ -98,6 +98,42 @@ autoComplete.SelectionMode = AutoCompleteSelectionMode.Multiple;
...
+</editors:SfAutoComplete>
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AutoComplete control with auto-append UI as TextWithSelection](Selection_images/winui-AutoComplete-TextWithSelection.png)
+
+### Auto-append UI as Text
+
+When entering text in the text box selection area, if the `AppendType` property is `Text`, the appended text can be text alone.
```
[the remark this answers is not in the document]
Comment (YuvarajMasilamani) [reply winui/AutoComplete/Selection.md:123]: Updated

File: winui/AutoComplete/Selection.md [hunk 2]
```diff
@@ -98,6 +98,42 @@ autoComplete.SelectionMode = AutoCompleteSelectionMode.Multiple;
 
 ![Multiple selection in WinUI AutoComplete](Selection_images/winui-autocomplete-multiple-selection.gif)
 
+## Auto-Append UI
+
+The AutoComplete control provides auto-append support with selection of text as well as text alone, as in Windows 11.The auto-append UI can be set by `AppendType` property.There are two different auto-append UI: `TextWithSelection` and `Text`.
```
[the remark this answers is not in the document]
Comment (YuvarajMasilamani) [reply winui/AutoComplete/Selection.md:103]: Updated

File: winui/AutoComplete/Selection.md [hunk 3]
```diff
@@ -98,6 +98,42 @@ autoComplete.SelectionMode = AutoCompleteSelectionMode.Multiple;
...
+
+### Auto-append UI as TextWithSelection
+
+When entering text in the text box selection area, if the `AppendType` property is `TextWithSelection`, the appended text can be the selection of text.
```
[the remark this answers is not in the document]
Comment (YuvarajMasilamani) [reply winui/AutoComplete/Selection.md:107]: Updated

File: winui/AutoComplete/Selection.md [hunk 4]
```diff
@@ -98,6 +98,40 @@ autoComplete.SelectionMode = AutoCompleteSelectionMode.Multiple;
 
 ![Multiple selection in WinUI AutoComplete](Selection_images/winui-autocomplete-multiple-selection.gif)
 
+## Auto-Append UI
+
+The AutoComplete control provides auto-append support with the selection of text as well as text alone. The auto-append UI can be defined by using the `AppendType` property. There are two types of auto-append UI: `TextWithSelection` and `Text`. When the auto-append UI is set to `Text`, the appended text appears with a faded foreground, similar to Windows 11.
```
Comment (Chrispineagunja) [inline winui/AutoComplete/Selection.md:103]: The AutoComplete control provides auto-append support with the selection of text as well as text alone. The auto-append UI can be defined by using the `AppendType` property. There are two types of auto-append UI: the `TextWithSelection` and `Text.` When the auto-append UI is set to `Text,` the appended text appears with a faded foreground, similar to Windows 11.

File: winui/AutoComplete/Selection.md [hunk 5]
```diff
@@ -98,6 +98,40 @@ autoComplete.SelectionMode = AutoCompleteSelectionMode.Multiple;
...
+
+### Auto-append UI as TextWithSelection
+
+When entering text in the text box selection area, if the `AppendType` property is `Text`, the appended text appears with fade foreground.
```
Comment (Chrispineagunja) [inline winui/AutoComplete/Selection.md:107]: When entering a text in the text box selection area, if the `AppendType` property is `Text,` the appended text appears with the faded foreground.

File: winui/AutoComplete/Selection.md [hunk 6]
```diff
@@ -98,6 +98,40 @@ autoComplete.SelectionMode = AutoCompleteSelectionMode.Multiple;
...
+</editors:SfAutoComplete>
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AutoComplete control with auto-append UI as TextWithSelection](Selection_images/winui-autocomplete-auto-append-textwithselection.png)
+
+### Auto-append UI as Text
+
+When entering text in the text box selection area, if the `AppendType` property is `Text`, the appended text appears in a fading foreground.
```
Comment (Chrispineagunja) [inline winui/AutoComplete/Selection.md:122]: When entering a text in the text box selection area, if the `AppendType` property is `Text,` the appended text appears in a fading foreground.

File: winui/ComboBox/Selection.md [hunk 7]
```diff
@@ -244,6 +244,40 @@ The selected items can be changed interactively by using keyboard or by selectin
 
 N> You can refer more information about customization of `ComboBoxTokenItem` from [this](https://help.syncfusion.com/winui/combobox/ui-customization#styling-comboboxtokenitem) link.
```
Comment (Chrispineagunja) [inline winui/ComboBox/Selection.md:245]: N> Refer to more information about customization of the `ComboBoxTokenItem` from [this](https://help.syncfusion.com/winui/combobox/ui-customization#styling-comboboxtokenitem) link.

File: winui/ComboBox/Selection.md [hunk 8]
```diff
@@ -244,6 +244,40 @@ The selected items can be changed interactively by using keyboard or by selectin
...
 
+## Auto-Append UI
+
+The ComboBox control provides auto-append support with the selection of text as well as text alone. The auto-append UI can be defined by using the `AutoAppendType` property. There are two types of auto-append UI: `TextWithSelection` and `Text`. When the auto-append UI is set to `Text`, the appended text appears with a faded foreground, similar to Windows 11.
```
Comment (Chrispineagunja) [inline winui/ComboBox/Selection.md:249]: The ComboBox control provides auto-append support with the selection of text as well as text alone. The auto-append UI can be defined by using the `AutoAppendType` property. There are two types of auto-append UI: The `TextWithSelection` and `Text,` When the auto-append UI is set to `Text,` the appended text appears with a faded foreground, similar to Windows 11.

File: winui/ComboBox/Selection.md [hunk 9]
```diff
@@ -244,6 +244,40 @@ The selected items can be changed interactively by using keyboard or by selectin
...
+
+### Auto-append UI as TextWithSelection
+
+When entering text in the text box selection area, if the `AutoAppendType` property is `Text`, the appended text appears with fade foreground.
```
Comment (Chrispineagunja) [inline winui/ComboBox/Selection.md:253]: When entering a text in the text box selection area, if the `AutoAppendType` property is `Text,` the appended text appears with the faded foreground.

File: winui/ComboBox/Selection.md [hunk 10]
```diff
@@ -244,6 +244,40 @@ The selected items can be changed interactively by using keyboard or by selectin
...
+</editors:SfComboBox>
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI ComboBox control with auto-append UI as TextWithSelection](Selection_images/winui-combobox-auto-append-textwithselection.png)
+
+### Auto-append UI as Text
+
+When entering text in the text box selection area, if the `AutoAppendType` property is `Text`, the appended text appears in a fading foreground.
```
Comment (Chrispineagunja) [inline winui/ComboBox/Selection.md:268]: When entering a text in the text box selection area, if the `AutoAppendType` property is `Text,` the appended text appears in a fading foreground.

File: winui/ComboBox/Selection.md (hunk 8 above)
Comment (YuvarajMasilamani) [reply winui/ComboBox/Selection.md]: Updated

File: winui/ComboBox/Selection.md (hunk 10 above)
Comment (YuvarajMasilamani) [reply winui/ComboBox/Selection.md]: Updated

File: winui/AutoComplete/Selection.md (hunk 4 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/Selection.md]: Updated

File: winui/AutoComplete/Selection.md (hunk 5 above)
Comment (YuvarajMasilamani) [reply winui/AutoComplete/Selection.md]: Updated

File: winui/ComboBox/Selection.md (hunk 7 above)
Comment (YuvarajMasilamani) [reply winui/ComboBox/Selection.md]: Updated

File: winui/ComboBox/Selection.md (hunk 9 above)
Comment (YuvarajMasilamani) [reply winui/ComboBox/Selection.md]: Updated

**Comment by YuvarajMasilamani** [comment]
rebuild

**Review by jeganraj-m — APPROVED** [review]

**Pull request merged**